components/arm/arm.go

```diff
@@ -107,15 +108,19 @@ func CreateStatus(ctx context.Context, a Arm) (*pb.Status, error) {
 		return nil, err
 	}
 	model := a.ModelFrame()
-	endPosition, err := motionplan.ComputeOOBPosition(model, jointPositions)
-	if err != nil {
-		return nil, err
+	var endPosition *v1.Pose
+	if jointPositions != nil && model != nil {
```

Comment:
Don't need to check for this here if you are also checking for it inside the function now